netstandard/ref/System.Security.Cryptography.X509Certificates.cs

```diff
@@ -86,15 +121,19 @@ public partial class X509Certificate : System.IDisposable, System.Runtime.Serial
     {
         public X509Certificate() { }
         public X509Certificate(byte[] data) { }
+        [System.CLSCompliantAttribute(false)]
```

Comment:
It's interesting to see this added everywhere there's a SecureString parameter, presumably because SecureString isn't CLS-compliant.

Is there a good guide to the impact of CLS compliance in the modern world? What benefit does adding this attribute have for .NET Standard? (This commit would certainly be smaller without it.)

I strongly suspect there's a good reason for just applying it everywhere it's already applied in the framework, but it would be good to know what that reason is.

Reply:
The patch was crafted mechanically by comparing .NET Standard with .NET Core 2.x. It seems when we added `SecureString` (b6cf07a9ce98bff92335d8f2b47f7e2d2d906287) we accidentally marked it as `[CLSCompliant(false)]` which we fixed when moving the type (edac8aa25cf1bf79f61bcac87b45f06d3b74c456). However, we didn't update the `[CLSCompliant(false)]` attributes the compiler forced on us since then. I've filed an issue to review `[CLSCompliant(..)]` in .NET Standard with .NET Framework/.NET Core and fix the incorrect attribution.

Reply:
> Is there a good guide to the impact of CLS compliance in the modern world?

We still care about CLS compliance.